### ISSUE TITLE
[v4 css] inverted table styling

### ISSUE THREAD

**Issue by Mottie** [twbs/bootstrap#21585]
When inverting a table style using the `table-inverse` class ([demo](http://codepen.io/Mottie/pen/OWVgYZ)):

1. **Striping**<br>
   With a "table-striped" class, the stripes are not discernibly different. In the css, the definition is the same as for the non-inverted row styling:

   ```css
   .table-striped tbody tr:nth-of-type(odd) {
     background-color: rgba(0, 0, 0, .05); /* $table-bg-accent; */
   }
   ```

2. Borders<br>
   The border styling of the non-inverted tables is a very subtle, using a border color of `#eceeef`. On inverted tables, we see `#fff`! It's way to bright!

   ```css
   .table-inverse td,
   .table-inverse th,
   .table-inverse thead th {
     border-color: #fff; /* $body-bg */
   }
   ```

3. Hover<br>
   Hovering isn't as bad when no stripe is applied to the table; but when a stripe is applied, the hover is only slightly more discernible than the row color ([demo](http://codepen.io/Mottie/pen/vgOJOW))!

   ```css
   .table-hover tbody tr:hover {
     background-color: rgba(0, 0, 0, .075) /* $table-bg-hover */
   }
   ```

This may not seem like a big deal to some of you young 'uns out there, but for elderly and visually impaired users, it would make a difference.

**Comment by LarryAzevedo** [comment]
I agree with @Mottie.

**Regarding the striping**, maybe we should change `$table-bg-accent: rgba($black,.05)` to `$table-bg-accent: rgba($gray-light,.075)`.

**Regarding the borders**, maybe we need a new variable called `$table-inverse-border-color` and set it's default value to `lighten($table-inverse-bg, 15%)`.

**Regarding the hover**, maybe we should change `$table-bg-hover: rgba($black,.075)` to `$table-bg-hover: rgba($gray-light,.075)`.

I tested these new settings and it looked good (on two completely different monitors) but would appreciate your feedback.  I also experimented with changing `$table-inverse-bg` to different values and that worked out well too.

So to recap, I'm proposing the following changes in _variables.scss (under the `$table-inverse-color` variable):
```
$table-inverse-border-color:    lighten($table-inverse-bg, 15%) !default;

$table-bg-accent:               rgba($gray-light,.075) !default;
$table-bg-hover:                rgba($gray-light,.075) !default;
```

And the following changes in _tables.scss:
```
.table-inverse {
  color: $table-inverse-color;
  background-color: $table-inverse-bg;

  th,
  td,
  thead th {
    border-color: $table-inverse-border-color;
  }

  &.table-bordered {
    border: 0;
  }
}
```
What do you guys think?

**Comment by mdo** [comment]
Fixed by #22217.
